wct.conf.json

```diff
@@ -3,8 +3,7 @@
   "plugins": {
     "local": {
       "browsers": [
-        "chrome",
-        "firefox"
```

Comment:
All the rest of our components only test in Firefox on local... just aligning this one.